### PR TITLE
Better solution for PowerPerturbation type instability

### PR DESCRIPTION
**Pull request by janlisse**

### CODE REVIEW

File: test/simulations/PowerPerturbation.jl [hunk 1]
```diff
@@ -10,8 +10,38 @@ using PowerDynamics: PowerPerturbation, simulate, SwingEqLVS, StaticLine, PowerG
...
-        tspan_fault = (0.1,1)),
+        tspan_fault = (0.1,1),
+        power_symbol = :P),
         grid, state, (0., 1.))
     @test sol != nothing
     @test sol.dqsol.retcode == :Success
 end
+
+@testset "test PowerPerturbation simulation invalid P should throw PowerPerturbationError" begin
+    invalidP = 1
```
Comment (SabineAuer) [inline test/simulations/PowerPerturbation.jl]: @janlisse: do you think we can't simply convert from integer to float? Isn't it kind of unhandy for users to be restricted to floats? Also, we would have to clearly document at every node type to use floats only.

File: src/simulations/PowerPerturbation.jl [hunk 2]
```diff
@@ -14,16 +14,34 @@ struct PowerPerturbation
     node_number
     fraction
     tspan_fault
+    power_symbol
```
Comment (SabineAuer) [inline src/simulations/PowerPerturbation.jl]: shouldn't the power_symbol with ":P" be default for a power perturbation? I would not ask for this as an input of the PowerPerturbation struct but make this a default. What do you think?
The if statement in line 30 I would definitely keep to check whether active power is part of the node's field names

File: test/simulations/PowerPerturbation.jl (hunk 1 above)
Comment (SabineAuer) [inline test/simulations/PowerPerturbation.jl]: @janlisse: do you think we can't simply convert from integer to float? Isn't it kind of unhandy for users to be restricted to floats? Also, we would have to clearly document at every node type to use floats only.

File: src/simulations/PowerPerturbation.jl (hunk 2 above)
Comment (janlisse) [reply src/simulations/PowerPerturbation.jl]: Good point, i made :P default.

File: test/simulations/PowerPerturbation.jl (hunk 1 above)
Comment (janlisse) [reply test/simulations/PowerPerturbation.jl]: You are right. I changed the code to convert Int to Float internally. This is much nicer from a user perspective now.

File: src/simulations/PowerPerturbation.jl (hunk 2 above)
Comment (SabineAuer) [reply src/simulations/PowerPerturbation.jl]: great, looks good now!

File: test/simulations/PowerPerturbation.jl (hunk 1 above)
Comment (SabineAuer) [reply test/simulations/PowerPerturbation.jl]: Thank you. This is much more handy now.

**Pull request merged**